packit.yml

```diff
@@ -0,0 +1,29 @@
+---
+# https://packit.dev/docs/configuration/
+
+specfile_path: simple-term.spec
+files_to_sync: [simple-term.spec, packit.yml]
```

Comment:
You don't need this, that's the [default](https://packit.dev/docs/configuration/#files_to_sync)

Reply:
https://github.com/mh21/simple-term/pull/11